### PR TITLE
feat(ai): introducing Qwen-2.5-coder

### PR DESCRIPTION
**Pull request by tgenaitay**
### Your checklist for this pull request

- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

#### Managed Inference
- Reference page with GPU config
- Navigation menu

#### Generative APIs
- How to about Continue's config
- Bring Qwen to supported models

### CODE REVIEW

File: ai-data/managed-inference/reference-content/qwen2.5-coder-32b-instruct.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## Compatible Instances
+
+| Instance type  | Max context length |
+| ------------- |-------------|
+| H100      | 128k (INT8)
+| H100-2      | 128k (INT8)
+
+## Model introduction
+
+Qwen2.5-coder is your intelligent programming assistant familiar more than 40 programming languages.
```
Comment (ldecarvalho-doc) [inline ai-data/managed-inference/reference-content/qwen2.5-coder-32b-instruct.mdx:40]:
```suggestion
Qwen2.5-coder is your intelligent programming assistant familiar with more than 40 programming languages.
```

File: ai-data/managed-inference/reference-content/qwen2.5-coder-32b-instruct.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+With Qwen2.5-coder deployed at Scaleway, your company can benefit from code generation, AI-assisted code repair, and code reasoning. 
+
+## Why is it useful?
+
+- Qwen2.5-coder achieved the best performance on multiple popular code generation benchmarks (EvalPlus, LiveCodeBench, BigCodeBench), outranking many open-source models and giving competitive performance with GPT-4o.
```
Comment (ldecarvalho-doc) [inline ai-data/managed-inference/reference-content/qwen2.5-coder-32b-instruct.mdx:45]:
```suggestion
- Qwen2.5-coder achieved the best performance on multiple popular code generation benchmarks (EvalPlus, LiveCodeBench, BigCodeBench), outranking many open-source models and providing competitive performance with GPT-4o.
```

File: ai-data/generative-apis/how-to/query-code-models.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+Scaleway's Generative APIs service allows users to interact with powerful code models hosted on the platform.
+
+Code models are inherently language models specialized in *understanding code*, *generating code* and *fixing code*. 
+
+As such, they will be available through the same interfaces as language models:
+- The Scaleway [console](https://console.scaleway.com) provides complete [playground](/ai-data/generative-apis/how-to/query-language-models/#accessing-the-playground), aiming to test models, adapt parameters, and observe how these changes affect the output in real-time.
+- Via the [Chat API](/ai-data/generative-apis/how-to/query-language-models/#querying-language-models-via-api)
+For more information on how to query language models, find [our dedicated documentation]((/ai-data/generative-apis/how-to/query-language-models/).
```
Comment (ldecarvalho-doc) [inline ai-data/generative-apis/how-to/query-code-models.mdx:21]:
```suggestion
For more information on how to query language models, find [our dedicated documentation](/ai-data/generative-apis/how-to/query-language-models/).
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: ai-data/generative-apis/how-to/query-code-models.mdx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  paragraph: Learn how to interact with powerful language models specialized in code using Scaleway's Generative APIs service.
+tags: generative-apis ai-data language-models code-models chat-completions-api
+dates:
+  validation: 2024-12-08
+  posted: 2024-12-08
+---
+
+Scaleway's Generative APIs service allows users to interact with powerful code models hosted on the platform.
+
+Code models are inherently language models specialized in *understanding code*, *generating code* and *fixing code*.
```
Comment (nerda-codes) [inline ai-data/generative-apis/how-to/query-code-models.mdx:16]:
```suggestion
Code models are inherently language models specialized in **understanding code**, **generating code** and **fixing code**. 
```

**Review by nerda-codes — APPROVED** [review]

File: ai-data/managed-inference/reference-content/qwen-2.5-coder-32b-instruct.mdx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+-H "Content-Type: application/json" \
+--request POST \
+--url "https://<Deployment UUID>.ifr.fr-par.scaleway.com/v1/chat/completions" \
+--data '{"model":"qwen/qwen2.5-coder-32b-instruct:int8", "messages":[{"role": "system", "content": "You are Qwen, created by Alibaba Cloud. You are a helpful code assistant."},{"role": "user","content": "Write a quick sort algorithm."}], "max_tokens": 1000, "temperature": 1, "stream": false}'
+```
+
+Make sure to replace `<IAM API key>` and `<Deployment UUID>` with your actual [IAM API key](/identity-and-access-management/iam/how-to/create-api-keys/) and the Deployment UUID you are targeting.
+
+<Message type="tip">
+  The model name allows Scaleway to put your prompts in the expected format.
```
Comment (scwgoire) [inline ai-data/managed-inference/reference-content/qwen-2.5-coder-32b-instruct.mdx]: Not sure there is added value here, what did you want to express?

**Pull request closed without merging**